db/seeds.rb

```diff
@@ -5,3 +5,4 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+recipes = Recipe.create([{ title: 'Ensaladilla Rusa'}, { title: 'Puré de verduras'}, { title: 'Taboulé con calabaza'}, { title: 'Ternera a la cerveza'}, { title: 'Pollo a la coca-cola'}, { title: 'Espaghettis carbonara'}, { title: 'Pizza margarita'}, { title: 'Callos a la riojana'}, { title: 'Rape al estilo cabañal'}, { title: 'Bacalao al pil-pil'}, { title: 'Pollo al chilindrón'}, { title: 'Pulpo a la gallega'}, { title: 'Tortilla de patata'}, { title: 'Fabada asturiana'}])
```

Comment:
- Por una cuestión de estilo y legibilidad se deben evitar las lineas largas. Por favor, usa saltos de linea y tabulaciones para organizar ese array (un objeto por linea).
- Conviene intentar ser constante con el estilo. Dejar un espacio tras la llave de apertura y no dejarla antes de la llave de cierre no queda muy bonito ;)
- También veréis que cuando se crean nuevos archivos suelen incluir un salto de linea al final. Ese símbolo que hay al final indica que no hay un salto de linea.